tools/arm_compiler/build_raspberry_pi.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+set -e
+
+# To be able to build any of the python packages, it is assumed that
+# you have a python directory in /usr/include/arm-linux-gnueabihf/.
+
+# To install the architecture includes for python on ubuntu trusty;
+# run:
+#  sudo dpkg --add-architecture armhf
+#  echo "deb [arch=armhf] http://ports.ubuntu.com/ trusty main universe" \
+#    sudo tee -a /etc/apt/sources.list.d/armhf.list
+#  # Ignore errors about missing armhf packages in other repos.
+#  sudo apt-get update
+#  sudo apt-get install libpython-all-dev:armhf
```

Comment:
Interesting.  I updated the comment to suggest using aptitude instead.

It did work for you in the end though?